src/Model/GroundWaterFlow/gwf3chd8.f90

```diff
@@ -245,56 +223,51 @@ subroutine chd_ck(this)
     character(len=*), parameter :: fmtchderr = &
       "('CHD BOUNDARY ',i0,' HEAD (',g0,') IS LESS THAN CELL &
       &BOTTOM (',g0,')',' FOR CELL ',a)"
-! ------------------------------------------------------------------------------
     !
     ! -- check stress period data
     do i = 1, this%nbound
       node = this%nodelist(i)
       bt = this%dis%bot(node)
       ! -- accumulate errors
-      if (this%head(i) < bt .and. this%icelltype(node) /= 0) then
+      if (this%head_mult(i) < bt .and. this%icelltype(node) /= 0) then
         call this%dis%noder_to_string(node, nodestr)
         write (errmsg, fmt=fmtchderr) i, this%head(i), bt, trim(nodestr)
```

Comment:
The error message should probably report the value that failed, head_mult() in this case.

Reply:
Addressed this concern in #1424.